Harvard Caselaw Access Project.js

```diff
@@ -0,0 +1,175 @@
+{
+	"translatorID": "2a1cafb9-6f61-48d3-b621-c3265fde9eba",
+	"label": "Harvard Caselaw Access Project",
+	"creator": "Franklin Pezzuti Dyer",
+	"target": "^https://(.*\\.)*case\\.law",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2024-01-15 23:41:24"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2024 Franklin Pezzuti Dyer
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, _url) {
+	if (doc.querySelectorAll('.case-container').length > 0) {
+		return 'case';
+	}
+	else if (getSearchResults(doc, true)) {
+		return 'multiple';
+	}
+	return false;
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = doc.querySelectorAll('div.result-title > div > a');
+	for (let row of rows) {
+		let href = row.href;
+		Z.debug(href);
```

Comment:
comment out all but the most essential debug statements. Most translators won't have any.